apps/localizer/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+# Install vgl + whatever is necessary for chorus use
+COPY ./core /tmp/core_scripts
+RUN cd /tmp/core_scripts/utilities && \
+    ./chorus-utils.sh && \
+    cd .. && mv entrypoint/docker-entrypoint.sh / && \
+    mv entrypoint/1-create-user.sh /docker-entrypoint.d && \
+    mv entrypoint/2-run-app.sh /docker-entrypoint.d
```

Comment:
I still cannot wrap my head around the idea that this is better than those base images from v1.

Reply:
@floriansipp can this be done at the end of the Dockerfile? If we change something, we don't want to have to rebuild all layers after.
This could be simplified like moving all scripts at once?

@greut The base images prevented to use already dockerized apps, the idea I had was to not add this to the Dockerfile itself but to another Dockerfile on top of it, which as you had said adds CI complexity, but then again prevents the use of already Dockerized apps... Let's consider this a beginning, we have to test the pros and cons. Base images had many issues and I'd like to move to a better solution for easier integration of apps.

Reply:
@iDmple i could change it to move all script at once (except the entry point that is one level higher) but at the moment i'd prefer to let it like that , just in case there is some extra script that appears at a later date for a edge case. If that's not the case then we'll simplify it to the max but as you say this is a beginning

Reply:
Done, until we do it otherwise